msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md

```diff
@@ -120,23 +120,23 @@ Teams AI Library offers an  integrated fact-checking system to tackle bot halluc
 
 The following table lists the updates to the Teams AI library:
 
-|Type |Description  |JavaScript|C#|
-|---------|---------|||
-|OpenAIModel    |The OpenAIModel class lets you call both OAI and Azure OAI with one single component. New models can be defined for other model types like LLaMA2.              | ✔️ |✔️|
-|Embeddings    | The OpenAIEmbeddings class lets you generate embeddings using either OAI or Azure OAI. New embeddings can be defined for things like OSS Embeddings.        | ✔️ |❌|
-|Prompts     | A new object-based prompt system enables better token management and reduces the likelihood of overflowing the model's context window.        | ✔️ |✔️|
-| Augmentation     | Augmentations simplify prompt engineering tasks by letting the developer add named augmentations to their prompt. Only  `functions`, `sequence`, and `monologue` style augmentations are supported.        | ✔️ |✔️|
-|Data Sources     |  A new DataSource plugin makes it easy to add RAG to any prompt. You can register a named data source with the planner and then specify the name[s] of the data sources they wish to augment the prompt.      | ✔️ |❌|
+|Type |Description  |.NET|JavaScript|Python|
+|---------|---------||||
+|OpenAIModel    |The OpenAIModel class lets you call both OAI and Azure OAI with one single component. New models can be defined for other model types like LLaMA2.              | ✔️ |✔️|✔️|
+|Embeddings    | The OpenAIEmbeddings class lets you generate embeddings using either OAI or Azure OAI. New embeddings can be defined for things like OSS Embeddings.        | ❌ |✔️|✔️|
+|Prompts     | A new object-based prompt system enables better token management and reduces the likelihood of overflowing the model's context window.        | ✔️ |✔️|✔️|
+| Augmentation     | Augmentations simplify prompt engineering tasks by letting the developer add named augmentations to their prompt. Only  `functions`, `sequence`, and `monologue` style augmentations are supported.        | ✔️ |✔️|✔️|
+|Data Sources     |  A new DataSource plugin makes it easy to add RAG to any prompt. You can register a named data source with the planner and then specify the name[s] of the data sources they wish to augment the prompt.      | ❌ |✔️|✔️|
 
 ## Code samples
 
-|Sample name | Description |.NET | Node.js |
-|----------------|-----------------|--------------|--------------|
-|Echo bot| This sample shows how to incorporate a basic conversational flow into a Microsoft Teams application using Bot Framework and the Teams AI library. | [View](https://github.com/microsoft/teams-ai/tree/main/dotnet/samples/01.messaging.echoBot) | [View](https://github.com/microsoft/teams-ai/tree/main/js/samples/01.messaging.a.echoBot) |
+|Sample name | Description |.NET | Node.js | Python|
```

Comment:
In addition to Echo bot, please update so we have checkmarks next to these 2 bots for Python:

- Message extensions: GPT-ME
- Light bot

Reply:
Updated.